data_steward/cdr_cleaner/cleaning_rules/deid/concept_suppression.py

```diff
@@ -46,8 +46,8 @@ def get_tables_in_dataset(client: Client, project_id, dataset_id,
                                              table_names=table_names))
     result = query_job.result()
 
-    if hasattr(result, 'errors') and result.errors:
```

Comment:
`query_jub.result` could throw GoogleCloudError and TimeoutError, we should put it into a try-catch block